src/Files.App/Helpers/Environment/ElevationHelpers.cs

```diff
@@ -19,5 +18,11 @@ public static bool IsElevationRequired(string filePath)
 
 			return _IsElevationRequired(filePath);
 		}
+
+		public static bool IsAppRunAsAdmin()
+		{
+			using WindowsIdentity identity = WindowsIdentity.GetCurrent();
+			return new WindowsPrincipal(identity).IsInRole(WindowsBuiltInRole.Administrator);
```

Comment:
It turns out that it's not specific to admin, the crash can happen even without admin if the app is running with high trust #13394.